tests/scene/test_decal.h

```diff
@@ -0,0 +1,134 @@
+/*  test_decal.h                                                          */
```

Comment:
```suggestion
/**************************************************************************/
/*  test_decal_3d.h                                                       */
```

Reply:
That was a rough start for my first contribution. I am very sorry I could not follow the simple rules... I will try to make sure they do not happen in the future.

Reply:
No worries! It can be hard to tell what the suggestion says at times, easiest just to copy the text directly